EHR_Purchasing/resources/schemas/dbscripts/postgresql/ehr_purchasing-20.000-20.001.sql

```diff
@@ -0,0 +1,176 @@
+(
+    rowId       serial,
+    attentionTo varchar(500),
+    city        varchar(100),
+    state       varchar(100),
+    country     varchar(100),
+    zip         varchar(15),
+    phoneNumber varchar(50),
+    email       varchar(500),
+
+    LSID        LSIDtype,
+    container   ENTITYID NOT NULL,
+    createdBy   USERID,
+    created     timestamp,
+    modifiedBy  USERID,
+    modified    timestamp,
+
+    CONSTRAINT PK_EHR_PURCHASING_SHIPPINGINFO PRIMARY KEY (rowId),
+    CONSTRAINT FK_EHR_PURCHASING_SHIPPINGINFO_CONTAINER FOREIGN KEY (Container) REFERENCES core.Containers (EntityId)
+);
+
+CREATE INDEX IDX_EHR_PURCHASING_SHIPPINGINFO_CONTAINER ON ehr_purchasing.shippingInfo (Container);
+
+-- line item unit, ex. EA = Each, BX = Box, etc.
+CREATE TABLE ehr_purchasing.itemUnits
+(
+    rowId           serial,
+    itemUnit            varchar(20),
+    unitDescription varchar(200),
+
+    LSID            LSIDtype,
+    container       ENTITYID NOT NULL,
+    createdBy       USERID,
+    created         timestamp,
+    modifiedBy      USERID,
+    modified        timestamp,
+
+    CONSTRAINT PK_EHR_PURCHASING_ITEM_UNITS PRIMARY KEY (rowId),
+    CONSTRAINT FK_EHR_PURCHASING_ITEM_UNITS_CONTAINER FOREIGN KEY (Container) REFERENCES core.Containers (EntityId)
+);
+
+CREATE INDEX IDX_EHR_PURCHASING_ITEM_UNITS_CONTAINER ON ehr_purchasing.itemUnits (Container);
+
+-- users associated with ehr_billing.aliases account
+CREATE TABLE ehr_purchasing.userAccountAssociations
+(
+    userId     int NOT NULL,
+    account    varchar(200) NOT NULL,
+
+    LSID       LSIDtype,
+    container  ENTITYID NOT NULL,
+    createdBy  USERID,
+    created    timestamp,
+    modifiedBy USERID,
+    modified   timestamp,
+
+    CONSTRAINT UQ_EHR_PURCHASING_USER_ACCT_ASSOCIATIONS UNIQUE (userId, account),
+    CONSTRAINT FK_EHR_PURCHASING_USER_ACCT_ASSOCIATIONS_CONTAINER FOREIGN KEY (Container) REFERENCES core.Containers (EntityId)
+);
+
+CREATE INDEX IDX_EHR_PURCHASING_USER_ACCT_ASSOCIATIONS_CONTAINER ON ehr_purchasing.userAccountAssociations (Container);
+
+-- line item status such as 'Ordered', 'Back ordered', 'Delivered' etc.
+CREATE TABLE ehr_purchasing.lineItemStatus
+(
+    rowId      serial,
+    status     varchar(100),
+
+    LSID       LSIDtype,
+    container  ENTITYID NOT NULL,
+    createdBy  USERID,
+    created    timestamp,
+    modifiedBy USERID,
+    modified   timestamp,
+
+    CONSTRAINT PK_EHR_PURCHASING_LINE_ITEM_STATUS PRIMARY KEY (rowId),
+    CONSTRAINT FK_EHR_PURCHASING_LINE_ITEM_STATUS_CONTAINER FOREIGN KEY (Container) REFERENCES core.Containers (EntityId)
+);
+
+CREATE INDEX IDX_EHR_PURCHASING_LINE_ITEM_STATUS_CONTAINER ON ehr_purchasing.lineItemStatus (Container);
+
+CREATE TABLE ehr_purchasing.purchasingRequests
+(
+    rowId            serial,
+    requestId        entityid,
+    vendorId         int,
+    account          varchar(200),
+    shippingInfoId   int,
+    justification    varchar(1000),
```

Comment:
Let's make these just "varchar" instead of varchar(1000). There's no harm in not having a cap.

Reply:
done